QuantConnect.TDAmeritrade/TDAmeritradeBrokerage.WebSocket.cs

```diff
@@ -648,9 +655,23 @@ private void HandleOrderCancelReplaceRequest(OrderCancelReplaceRequestMessage? o
                 return;
             }
 
-            leanOrder.BrokerId[0] = newBrokerageOrderKey;
+            var updateOrderEvent = new BrokerageOrderIdChangedEvent()
+            {
+                OrderId = leanOrder.Id,
+                BrokerId = new List<string> { newBrokerageOrderKey }
+            };
+
+            OnOrderIdChangedEvent(updateOrderEvent);
+
+            if (!TryGetLeanOrderById(newBrokerageOrderKey, out var newLeanOrder))
```

Comment:
don't copypaste the code;
also, if we enter return here, we will not Set() the _onUpdateOrderWebSocket event which is dangerous.